cheatsheets/Forgot_Password_Cheat_Sheet.md

```diff
@@ -15,16 +15,20 @@ The following short guidelines can be used as a quick reference to protect the f
 
 ## Content
 
-- [Forgot Password and MFA Cheat Sheet](#forgot-password-and-mfa-cheat-sheet)
```

Comment:
Personally I do not like this change because we have separate MFA CS and here we should only content relevant to Forgot Password and cross-reference MFA CS.

Reply:
Not really sure what at what you're pointing. I am guessing OTPs? It's a method. I did cross reference it, I had to discuss it though. It's barely a 2 liner description.